tests/unit/src/main/scala/tests/BaseScalaCliSuite.scala

```diff
@@ -297,6 +297,41 @@ abstract class BaseScalaCliSuite(scalaVersion: String)
         s"Expected no scalameta errors, got: $parserDiagnostics",
       )
 
+      _ <- server.didChangeConfiguration(
+        """{
+          |  "enable-semantic-highlighting": true
+          |}
+          |""".stripMargin
+      )
+      expected =
+        s"""|<<#!/usr/bin/env -S scala-cli shebang --java-opt -Xms256m --java-opt -XX:MaxRAMPercentage=80>>/*comment*/
+            |<<//> using scala "$scalaVersion">>/*comment*/
+            |<<//> using lib "com.lihaoyi::utest::0.7.9">>/*comment*/
+            |<<//> using lib "com.lihaoyi::pprint::0.6.4">>/*comment*/
+            |
+            |<<import>>/*keyword*/ <<foo>>/*class*/.<<Foo>>/*class*/
+            |<<import>>/*keyword*/ <<utest>>/*namespace*/.<<_>>/*variable,readonly*/
+            |
+            |<<pprint>>/*namespace*/.<<log>>/*method*/(<<2>>/*number*/) <<// top-level statement should be fine in a script>>/*comment*/
+            |
+            |<<object>>/*keyword*/ <<MyTests>>/*class*/ <<extends>>/*keyword*/ <<TestSuite>>/*class,abstract*/ {
+            |  <<pprint>>/*namespace*/.<<log>>/*method*/(<<2>>/*number*/)
+            |  <<val>>/*keyword*/ <<tests>>/*variable,definition,readonly*/ = <<Tests>>/*class*/ {
```

Comment:
I'm not sure what's happening, when `useBsp` is `false` it seems like it doesn’t  see symbols from `utest` (we get `NoSymbol` on them), but symbols from `pprint` are fine

Reply:
I think we might be having a race condition when using BSP. We get the semantic highlight before the bsp server actually manages to start